### ISSUE TITLE
Wrong tile shows text for mouse hover.

### ISSUE THREAD

**Issue by PRabahy** [evolvio/evolv.io#151]
If I hover over a cell when the map is rotated (because I'm follow a specific creature), the Yums info shows up for a different cell.

![image](https://cloud.githubusercontent.com/assets/429961/24387960/20ec515a-1346-11e7-8461-357caf7bfb5a.png)

**Comment by PRabahy** [comment]
Screen shot doesn't show it, but my mouse was over Ghanode when I took it.

**Comment by PRabahy** [comment]
Oh, its not that the map is rotated, it has something to do with the zoom or Z/Y translation.

**Comment by UnknownEvil** [comment]
![No problem](https://puu.sh/v4OWB/5372db033e.gif)

**Comment by PRabahy** [comment]
What did you use to make the gif? I think that would be clearer about what I'm seeing.

**Comment by UnknownEvil** [comment]
I'm using puush (ShareX) and then you just do right-click > capture > Screen Recording (GIF)

**Comment by UnknownEvil** [comment]
No, I was wrong, you're entirely correct about this.

**Comment by NateUniAccount** [comment]
Can you remember when it was ok or even better which PR changed the behaviour?

**Comment by PRabahy** [comment]
I don't remember if it was ever correct or not. I just noticed it randomly one day. Its not a big deal to me, but I figured I would at least record it.

**Comment by UnknownEvil** [comment]
I fixed it in my most recent pull request (but it comes with the other changes). 
I'll make a separate pull request fixing it, without the other changes.

**Comment by UnknownEvil** [comment]
Just for context, the bug appeared after #143 and I've fixed it in #156

**Comment by kodomazer** [comment]
Closing the issue since it's fixed.
